release-team/role-handbooks/release-notes/README.md

```diff
@@ -34,7 +34,7 @@ The Release Notes role is responsible for collecting and fine-tuning release-not
 
 ## Tools
 
-- [Release notes tool](https://github.com/marpaia/release-notes)
+- [Release notes tool](https://github.com/kubernetes/release/tree/master/toolbox/relnotes)
```

Comment:
This should be https://github.com/kubernetes/release/tree/master/cmd/release-notes.